ister_gui.py

```diff
@@ -2146,6 +2167,12 @@ def handler(self, config):
                         else:
                             if bundle in config['Bundles']:
                                 config['Bundles'].remove(bundle)
+
+        # update only if user had chance to unselect
+        if not any(bundle['name'] == 'sysadmin-basic'
+                   for bundle in self.required_bundles):
+            self.sysadmin_basic_state = 'sysadmin-basic' in config['Bundles']
```

Comment:
So config['Bundles'] holds the default_bundles. Now default_bundles may have had sysadmin-basic removed from it if there is a user created but sysadmin-basic would have been added to config['Bundles'] separately in that case (fun logic chain to follow). I may have missed it but I don't see a way for sysadmin-basic to get back into default_bundles if a user ended up changing selections nor do I understand how self.bundles gets reset back to a default state either.